Demo/ViewController.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+class CollectionCell: UICollectionViewCell {
```

Comment:
I was thinking of having the Unsplash thing _be_ the SweetUIKit demo app. In here.